src/pages/Chat/components/Message.tsx

```diff
@@ -1,38 +1,34 @@
 import { Component, Show } from "solid-js";
 import { ThumbsUp, ThumbsDown } from "lucide-solid";
-import showdown from "showdown";
-
 import MessageProps from "../../../models/Message";
-
 import "./Message.css";
-
-const showdownConverter = new showdown.Converter();
+import { mdConvert } from "../../../lib/convertMark";
 
 /** An individual message. */
+
 const Message: Component<MessageProps> = (props) => (
-  <span class="flex gap-4">
+  <span class={props.speaker.isHuman ? "flex gap-4 flex-row-reverse" : "flex gap-4"}>
```

Comment:
What does `flex-row-reverse` do here?

Reply:
made the user's chat go right, like some chat applications like telegram, iMessage. If you don't like it, I'll make it the way it was.